modules/devices/mmd.json

```diff
@@ -0,0 +1,2271 @@
+{
+    "@context": "\/api\/3\/contexts\/StagingModelMetadata",
+    "@id": "\/api\/3\/staging_model_metadatas\/6b855b7f-cc3a-4989-a581-61d1547dbe16",
+    "@type": "StagingModelMetadata",
+    "type": "devices",
+    "parentType": null,
+    "tableName": "device",
+    "ownable": false,
+    "userOwnable": false,
+    "queueable": false,
+    "trackable": false,
+    "taggable": true,
+    "peerReplicable": false,
```

Comment:
Some key settings are missing in all modules, as shown in the screenshot. Please confirm if the current settings work for you or configure the relevant settings.

![image](https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-framework/assets/46894140/035ebf4d-35e6-460e-a506-df8fe6b723cf)